pyproject.toml

```diff
@@ -60,7 +60,7 @@ torch = [
     "torch>=2.2.0;python_version>='3.12' and python_version<'3.13'",
     "torch>=2.6.0;python_version>='3.13'",
 ]
-pomegranate = ['pomegranate>=0.15,<1.0']
+pomegranate = ['pomegranate>=1.1.1,<2.0']
```

Comment:
We should make 1.1.2 our minimum dependency to ensure package metadata has the changes we asked for